aiohttp_session/__init__.py

```diff
@@ -104,6 +104,12 @@ def __delitem__(self, key):
 SESSION_KEY = 'aiohttp_session'
 STORAGE_KEY = 'aiohttp_session_storage'
 
+async def is_active_session(request):
```

Comment:
This function needs a docstring

Reply:
Also, the name `is_session_active` would sound better.

Reply:
Oh, and `async` is absolutely unnecessary because it's synchronous and doesn't use `await` anywhere.

Reply:
re: asynch - absolutely right... sorry... force of habit since so many of my aiohttp related functions have been async as of necessity. Can be removed.

I'm also fine with the rename `is_session_active` although I guess I'm thinking of the conditional construct 

```
if is_active_session(request):
   ...
```

which sounds a slightly more natural English than

```
if is_session_active(request):
 ...
```

Reply:
I think `is_active_session` sounds like the expectation is that you're checking that the argument is a session and it's weird to see a request there instead.

Reply:
> I think `is_active_session` sounds like the expectation is that you're checking that the argument is a session and it's weird to see a request there instead.

Yep, I'd read that in English as "Is this an active session?" vs. "Is the session currently active (in this request)?".

Without context, both sound natural.